### ISSUE TITLE
Saving Doesn't Reflect Change Until Web Refresh

### ISSUE THREAD

**Issue by njross33** [SeedCompany/cord-field#50]
A number of users have put in tickets ranging from, "I put in an translation end date and it doesn't show." As an admin user, if I go to that project, the end date does show. If you go to that user's machine and actually hit the refresh button (F5 on PC, Command+R on Mac), it will then show that the end date is there. This seems only to apply to the "Plan" area of a project, as I've heard users Products not saving (or showing after saving) and yet, it does once you do a browser refresh.

Basically I've been telling the users to refresh and they'll see their work did go through or I'll just verify that it's there and they are set. Just a bit of confusion on their end when they don't see the change reflected.

**Comment by sethmcknight** [comment]
closing out as we won't do this in v2. This should not be an issue in v3 but if it is after release we will revisit.
